open-api/rest-catalog-open-api.py

```diff
@@ -171,7 +171,6 @@ class SortOrder(BaseModel):
 
 class Summary(BaseModel):
     operation: Literal['append', 'replace', 'overwrite', 'delete']
-    additionalProperties: Optional[str] = None
```

Comment:
I would have expected the summary to also have `Optional[Dict[str, str]] = None`

Reply:
When I used the plain OpenAPI generator (tested on 6.6.0 & 7.0.0) to generate python code, this is what I got
```
...
class SnapshotSummary(BaseModel):
    """
    SnapshotSummary
    """
    operation: StrictStr = Field(...)
    additional_properties: Dict[str, Any] = {}
    __properties = ["operation"]
...
```
Which is closer to what you suggested above. I guess the code generation process here might leverage some different library.